mpmp/prediction/cross_validation.py

```diff
@@ -0,0 +1,279 @@
+"""
+Functions to split data for cross-validation.
```

Comment:
I will definitely take a look at these scripts in the future when I have to run any classificiation/regression task :)